### ISSUE TITLE
Can't install mailgun gem

### ISSUE THREAD

**Issue by lectricas** [mailgun/mailgun-ruby#60]
C:\Users\lectricas\Documents\obed18-menu>bundle install
Fetching gem metadata from https://rubygems.org/
Fetching version metadata from https://rubygems.org/
Could not find gem 'mailgun-ruby (~> 1.0.5) x64-mingw32' in any of the gem
sources listed in your Gemfile or available on this machine.

C:\Users\lectricas\Documents\obed18-menu>]

**Comment by cucho** [comment]
Same here. I just removed the version requirement and it installed v1.1.0

`gem 'mailgun-ruby', require: 'mailgun'`

**Comment by lectricas** [comment]
gem 'mailgun-ruby', '1.1.0', require: 'mailgun'

solved the issue

**Comment by solisoft** [comment]
It's KO for me

````
Resolving dependencies....
Bundler could not find compatible versions for gem "rest-client":
  In snapshot (Gemfile.lock):
    rest-client (= 2.0.0)

  In Gemfile:
    mailgun-ruby was resolved to 1.0.0, which depends on
      rest-client (~> 1.6)

    mailgun_rails was resolved to 0.8.0, which depends on
      rest-client (>= 1.6.7)

    stripe was resolved to 1.54.0, which depends on
      rest-client (< 4.0, >= 1.4)

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.
````

**Comment by pirogoeth** [comment]
Updating from this point forward should be less bad -- all following releases will follow semver.
